public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -88,96 +109,202 @@ const BannerTestVariantEditor: React.FC<BannerTestVariantEditorProps> = ({
-        label="Header"
-        margin="normal"
-        variant="outlined"
-        disabled={!editMode}
-        fullWidth
-      />
-
-      <TextField
-        inputRef={register({
-          required: EMPTY_ERROR_HELPER_TEXT,
-          validate: invalidTemplateValidator,
-        })}
-        error={errors.body !== undefined}
-        helperText={errors.body ? errors.body.message : BODY_DEFAULT_HELPER_TEXT}
-        onBlur={handleSubmit(onSubmit)}
-        name="body"
-        label="Body copy"
-        margin="normal"
-        variant="outlined"
-        multiline
-        rows={10}
-        disabled={!editMode}
-        fullWidth
-      />
-
-      {variant.template === BannerTemplate.ContributionsBanner && (
         <TextField
           inputRef={register({
+            required: EMPTY_ERROR_HELPER_TEXT,
             validate: invalidTemplateValidator,
           })}
-          error={errors.highlightedText !== undefined}
-          helperText={
-            errors.highlightedText ? errors.highlightedText.message : HIGHTLIGHTED_TEXT_HELPER_TEXT
-          }
+          error={errors.messageText !== undefined}
+          helperText={errors.messageText ? errors.messageText.message : BODY_DEFAULT_HELPER_TEXT}
           onBlur={handleSubmit(onSubmit)}
-          name="highlightedText"
-          label="Highlighted text"
+          name="messageText"
+          label="Body copy"
           margin="normal"
           variant="outlined"
+          multiline
+          rows={10}
           disabled={!editMode}
           fullWidth
         />
-      )}
 
+        {template === BannerTemplate.ContributionsBanner && (
+          <TextField
+            inputRef={register({
+              validate: invalidTemplateValidator,
+            })}
+            error={errors.highlightedText !== undefined}
+            helperText={
+              errors.highlightedText
+                ? errors.highlightedText.message
+                : HIGHTLIGHTED_TEXT_HELPER_TEXT
+            }
+            onBlur={handleSubmit(onSubmit)}
+            name="highlightedText"
+            label="Highlighted text"
+            margin="normal"
+            variant="outlined"
+            disabled={!editMode}
+            fullWidth
+          />
+        )}
+
+        <div className={classes.buttonsContainer}>
+          <Typography className={classes.sectionHeader} variant="h4">
+            {`Buttons${labelSuffix}`}
+          </Typography>
+
+          <VariantEditorButtonsEditor
+            primaryCta={content.cta}
+            secondaryCta={content.secondaryCta}
+            updatePrimaryCta={updatePrimaryCta}
+            updateSecondaryCta={updateSecondaryCta}
+            isDisabled={!editMode}
+            onValidationChange={onValidationChange}
+            supportSecondaryCta={true}
+          />
+        </div>
+      </div>
+    </>
+  );
+};
+
+const BannerTestVariantContentEditorWithStyles = withStyles(styles)(BannerTestVariantContentEditor);
```

Comment:
we could potentially refactor to use the `useStyles` way of styling instead of the `withStyles` to avoid having to do this (it's also generally a little nicer) e.g: https://github.com/guardian/support-admin-console/blob/fe9cf64d613a9d913a6e1b8f21a4d8fad285dc61/public/src/components/shared/liveSwitch.tsx#L4

Reply:
oh yes that is much nicer, thanks!